### DISCUSSION TITLE
Fundamentals - Image and Imaging

### DISCUSSION THREAD

**Discussion by bellabf** [huggingface/computer-vision-course#83] [01. Fundamentals]
Hello everyone, 
This is the revised outline for the  **Image and Imaging chapter**.

**1. What is an image? [image.mdx]**: taken by me and @seshu-pavan
- [x] Definition of Image
- [x] Difference between Image and Video
- [x] Image-specific challenges (i.e.comparing images to other types of data)

**2. Fundamentals of Imaging and its technical aspects of images [imaging.mdx]** taken by @krmdel
- [x] Image formation: Bayer Grid / Filter, Digital Image Sensing, Color space representations
- [x] Image representation: Type of image representations, RGB's, image as the matrix,
- [x] Output of images: Image resolution, Pixel/Voxel, Dimensions Channels, Dynamic range, Bit depth, File Compression, Image formats, Metadata
- [x] enhancement and restoration 

**3. How we imaged everything/When in doubt, image it. [extension-image.mdx]** taken by me 
- [x] Challenges in imaging real-life 
- [x] Imaging everything (how we extend sensors to make images from things that are not usually thought of as images)
- [x] Perspective on what we obtained from this and its impact on us (i.e. we have imaged outer space and electrons; we have an MRI machine capable of imaging inside of us without ever touching 🤯)
- [x] Example(s) of how image-acquired characteristics of images come from what types of sensors they record, challenges/difficulties in computer vision change according to the image/data, not all data is RGB

**4. Preprocessing for Computer Vision. [examples-preprocess.mdx]** taken by @mfidabel and @krmdel

- [x] pre-processing techniques to compare images types (i.e. histograms, thresholds techniques, edge detection look like in image coming from different conditions
- [x] common operations before training (downsampling and data augmentation).

Our biggest concern was the possible overlap of our content with the “Feature Detection/Feature Extraction chapter” content, especially regarding image transformations, filters, and convolution. We opt to left it out, considering that the [ proposed outline for Feature Detection.](https://github.com/johko/computer-vision-course/issues/61) and to stay in touch with their team. 

Taken in this case, just means we were the first people to start the chapter and/or we are focus on it. 
There are a lot of active discussions between members to make the content more cohesive and easy to follow. 

Let us know what you think!

Team members - @bellabf, @seshu-pavan, @mfidabel @froestiago, @krmdel
bcc - @johko @lunarflu @merveenoyan @MKhalusova

Best,
Image and Imaging team

**Comment by adamm-hf** [comment]
Great! Broadly looks very nice. ✨ 😄 

>Our biggest concern was the possible overlap of our content with the “Feature Detection/Feature Extraction chapter” content, especially regarding image transformations, filters, and convolution. We opt to left it out, considering that the https://github.com/johko/computer-vision-course/issues/61 and to stay in touch with their team.

I think it's good to leave out for now, if it needs to be added we can do so in a later update
